pom.xml

```diff
@@ -196,6 +184,13 @@
             <version>1.2.5</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.jodd</groupId>
+            <artifactId>jodd-util</artifactId>
```

Comment:
Had this previously been a dependency of `redission`?

Reply:
Absolutely.. that's for another PR. It looks like our sole usage of it is to treat a sequence of whitespace the same as null or empty.